scripts/build.d/pybind11

```diff
@@ -2,19 +2,45 @@
 
 set -e
 
-pkgname=pybind11
-pkgbranch=${VERSION:-master}
-pkgfull=$pkgname-$pkgbranch
+SRCSYNC=${SRCSYNC:-git}
```

Comment:
Default to use git.  Note that this is a different variable than `SYNCGIT` used before.